runtime/onert/core/include/backend/ITensorRegistry.h

```diff
@@ -110,6 +110,11 @@ template <typename T_Tensor> class PortableTensorRegistryTemplate : public ITens
 
   const ir::OperandIndexMap<std::shared_ptr<T_Tensor>> &managed_tensors() { return _managed; }
 
+  const ir::OperandIndexMap<std::shared_ptr<IPortableTensor>> &external_tensors()
```

Comment:
actually i'm confused this name `external_tensors` is right. `portable_tensors`? hmm.. but setter(`setExternalTensor`) uses the word `external`.